pkgs/development/libraries/libdmtx/default.nix

```diff
@@ -1,18 +1,27 @@
-{ lib, stdenv, fetchurl, pkg-config }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+}:
 
 stdenv.mkDerivation rec {
-  name = "libdmtx-0.7.4";
+  pname = "libdmtx";
+  version = "0.7.5";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/libdmtx/${name}.tar.bz2";
-    sha256 = "0xnxx075ycy58n92yfda2z9zgd41h3d4ik5d9l197lzsqim5hb5n";
+  src = fetchFromGitHub {
+    owner = "dmtx";
+    repo = "libdmtx";
+    rev = "v${version}";
+    sha256 = "0wk3fkxzf9ip75v8ia54v6ywx72ajp5s6777j4ay8barpbv869rj";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
 
   meta = {
     description = "An open source software for reading and writing Data Matrix barcodes";
-    homepage = "http://libdmtx.org";
+    homepage = "https://github.com/dmtx/libdmtx";
+    changelog = "https://github.com/dmtx/libdmtx/blob/v${version}/ChangeLog";
     platforms = lib.platforms.all;
     maintainers = [ ];
```

Comment:
Maybe you can add yourself as a maintainer?

Reply:
No thanks.